h/templates/client/viewer.html

```diff
@@ -40,7 +40,7 @@
       ng-mouseenter="focus(child.message)"
       ng-click="scrollTo(child.message)"
       ng-mouseleave="focus()"
-      ng-repeat="child in threadRoot.children | orderBy : sort.predicate"
+      ng-repeat="child in threadRoot.children | filter:notOrphan() | orderBy : sort.predicate"
```

Comment:
The docs is not entirely clear about this, so I though that I need to use parentheses so that it can recognize a function. But indeed, some testing reveals that I don't actually have to invoke it. Simplified.